packages/js-config-webpack-plugin/package.json

```diff
@@ -0,0 +1,81 @@
+{
+    "name": "js-config-webpack-plugin",
+    "version": "0.3.0",
+    "description": "",
+    "main": "src/index.js",
+    "license": "MIT",
+    "author": "Namics AG - Technology & Application",
+    "contributors": [
+        {
+            "name": "Jan Biasi",
+            "email": "biasijan@gmail.com",
+            "url": "https://github.com/janbiasi"
+        },
+        {
+            "name": "Matthias Giger",
+            "email": "matthias.giger@namics.com",
+            "url": "https://github.com/naminho"
+        },
+        {
+            "name": "Davide De Giorgio",
+            "email": "dave_degiorgio@hotmail.com",
+            "url": "https://github.com/degiorgioo"
+        },
+        {
+            "name": "Ernst Ammann",
+            "email": "ernscht@gmail.com",
+            "url": "https://github.com/ernscht"
+        },
+        {
+            "name": "Jan Nicklas",
+            "email": "j.nicklas@me.com",
+            "url": "https://github.com/jantimon"
+        }
+    ],
+    "engines": {
+        "node": ">=8 <11",
+        "npm": ">=5"
+    },
+    "scripts": {
+        "test": "jest --runInBand --verbose --forceExit",
+        "posttest": "tsc",
+        "test-watch": "jest --watch --verbose --forceExit",
+        "update-snapshots": "jest -u --runInBand"
+    },
+    "peerDependencies": {
+        "typescript": ">=2.0.0",
+        "webpack": ">=4.0.0",
+        "@babel/core": ">=7.0.0-rc.1",
+        "@babel/preset-env": ">=7.0.0-rc.1"
+    },
+    "devDependencies": {
+        "@types/jest": "23.3.1",
+        "@types/node": "10.5.7",
+        "@types/webpack": "^4.4.9",
+        "jest": "23.5.0",
+        "jsdom": "^11.12.0",
+        "rimraf": "2.6.2",
+        "sourcemap-validator": "1.1.0",
+        "typescript": "3.0.1",
+        "webpack": "^4.16.5"
+    },
+    "dependencies": {
+        "babel-loader": "^8.0.0-beta.4",
+        "thread-loader": "^1.2.0",
+        "@babel/core": "^7.0.0-rc.1",
```

Comment:
please move babel core and babel env to dev dependencies as they are already peer dependencies